pkgs/applications/video/vokoscreen/default.nix

```diff
@@ -1,21 +1,26 @@
-{ stdenv, fetchgit
-, pkgconfig, qtbase, qttools, qmake, qtx11extras, alsaLib, libv4l, libXrandr
+{ stdenv, fetchFromGitHub
+, pkgconfig, qtbase, qttools, qmake, qtmultimedia, qtx11extras, alsaLib, libv4l, libXrandr
 , ffmpeg
 }:
 
-stdenv.mkDerivation {
-  name = "vokoscreen-2.5.0";
-  src = fetchgit {
-    url = "https://github.com/vkohaupt/vokoscreen.git";
-    rev = "8325c8658d6e777d34d2e6b8c8bc03f8da9b3d2f";
-    sha256 = "1hvw7xz1mj16ishbaip73wddbmgibsz0pad4y586zbarpynss25z";
+stdenv.mkDerivation rec {
+
+  pname = "vokoscreen";
+  version = "2.5.8-beta";
+
+  src = fetchFromGitHub {
+    owner   = "vkohaupt";
+    repo    = "vokoscreen";
+    rev     = "2bf2469803f76a0de4abdd19f3abc3229b259b7e";
```

Comment:
Can you use the hit tag name here? It will make future updates easier.

Reply:
since the tag is identical to the package version, I use the version variable directly now